sys/arm64/include/armreg.h

```diff
@@ -79,6 +79,13 @@
 #define	CNTP_CTL_ISTATUS	(1 << 2)
 
 /* CPACR_EL1 */
+#if __has_feature(capabilities)
+#define	CPACR_CEN_MASK		(0x3 << 18)
+#define	 CPACR_CEN_TRAP_ALL1	(0x0 << 18) /* Traps from EL0 and EL1 */
+#define	 CPACR_CEN_TRAP_EL0	(0x1 << 18) /* Traps from EL0 */
+#define	 CPACR_CEN_TRAP_ALL2	(0x2 << 18) /* Traps from EL0 and EL1 */
+#define	 CPACR_CEN_TRAP_NONE	(0x3 << 18) /* No traps */
```

Comment:
Indenting seems a bit odd (but that seems indemic to this file so probably not a blocker.)

I do wonder if defining an upstreamable `CPACR_CEN_SHIFT` and avoiding the large ifdef would be cleaner.

Reply:
It should be the standard indentation for this file, the field values have an extra space to help find different fields visually.